README.md

```diff
@@ -1 +1,33 @@
-# prsh
\ No newline at end of file
+# prsh
+
+Commands:
+
+- npm dev: Start the development server
+- npm build: Build & clean the project
+- npm lint: Lint the project (Lint on save reccommended)
+
+
+
+## Project setup
+
+### Database setup
+1. Create a new PostgresSQL database image using Docker:
+```bash
+
+docker run --name prsh -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres
+
+```
+
+2. Add the following environment variables to a new `.env` file in the root of the project:
+```bash
+DATABASE_URL="postgresql://postgres:mysecretpassword@localhost:5432/prsh"
+```
+
+
+
+
+3. Create a new database called `prsh` by running Prisma's migrate command:
+```bash
+npx prisma migrate dev
+```
+
```

Comment:
Ban and I both had to run the command `docker run -p 5432:5432 -e POSTGRES_PASSWORD=mysecretpassword -d postgres`. It links localhost to port 5432 since we are hosting in Docker container, not our localhost. Not sure if another command indirectly does this and we forgot to run it.